spec/factories/flights.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :flight do
+    flight_no 43
+    airline "Emirate Airline"
+    origin_airport_id 435
+    destination_airport_id 76
+    departure_datetime Time.now
+    arrival_datetime Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.